crates/nu-command/tests/commands/path/basename.rs

```diff
@@ -81,3 +81,11 @@ fn replaces_basename_of_path_ending_with_double_dot() {
     let expected = join_path_sep(&["some/file.txt/..", "eggs"]);
     assert_eq!(actual.out, expected);
 }
+
+#[test]
+fn const_path_basename() {
+    let actual =
+        nu!(cwd: "tests", pipeline("const name = ('spam/eggs.txt' | path basename); $name"));
```

Comment:
regarding all the tests i think, we can have a simpler
```suggestion
        nu!("const name = ('spam/eggs.txt' | path basename); $name");
```
:yum:

Reply:
my bad @kubouch, i meant this could be applied to the other new tests as well, not only `const_path_basename` :open_mouth: 

i posted a single comment to avoid duplicates :relieved:

Reply:
I simplified more tests, should be good now.